background.ts

```diff
@@ -946,6 +946,7 @@ ipcMainProxy.handle('show-snapshots-blocking-dialog', async(
     if (channel === 'dialog/mounted') {
       if (os.platform() !== 'darwin') {
         mainWindow?.webContents.send('window/blur', true);
+        mainWindow?.setResizable(false);
```

Comment:
`setResizable()` is not related to the scope of this PR. I think we may want to make the window not resizable during the snapshots operations, to avoid the users to completely hide the window behind the snapshot's blocking dialog. Otherwise, I can simply remove it and address it in another issue/discussion.